### DISCUSSION TITLE
Dashboard page takes me to the home page & cant get data from PocketBase

### DISCUSSION THREAD

**Discussion by kenyonbowers** [pockethost/pockethost#138] [Q&A]
Hello. I have been using this for testing PocketBase and it was working yesterday. However, all the sudden, I can't access my dashboard because it keeps sending me to the home page. Also, when trying to go the the admin url for the PocketBase itself, I just get hit with an infinitely loading page. And I also cannot grab any data from the PocketBase either.

**Comment by TechStudent10** [comment]
Can confirm. Facing the same issue. Pinging @benallfree just so that he can see it.

**Comment by benallfree** [comment]
@kenyonbwrs @TechStudent10 First of all, thank you for using pockethost enough to even notice an outage! :) That is an amazing holiday gift for me.

I recently upgraded things to v0.5.6 and I suspect that has broken something for each of you.

Investigating now...

Would you mind sharing either here or with ben@pockethost.io, what subdomains are you trying to access?

**Reply by kenyonbowers** [reply]
I will send you an email

**Reply by TechStudent10** [reply]
Alright

**Comment by kenyonbowers** [comment]
Thank you spending your time to put them back online!

**Comment by RA9** [comment]
I don’t even have access to `pockethost` dashboard, and the `pockethost` homepage doesn’t show the login and signup links

**Reply by RA9** [reply]
![image](https://user-images.githubusercontent.com/20514920/209447733-6ac6e9d4-1b73-4a47-92df-535e3b57e277.png)

**Comment by TechStudent10** [comment]
Issue appears to be fixed on my end:

![image](https://user-images.githubusercontent.com/76978184/209447828-cda2f25f-ddbf-45a3-8df6-ad95f9d32c62.png)

![image](https://user-images.githubusercontent.com/76978184/209447839-97e51b04-6367-44d1-b707-135dfddd9ff5.png)

**Reply by RA9** [reply]
Thanks, @benallfree. It is fixed now.

**Reply by kenyonbowers** [reply]
Its fixed on my end as well

**Reply by TechStudent10** [reply]
Thank you @benallfree!

**Comment by benallfree** [comment]
Please continue to report stability issues in this thread. v0.5.6 just released today has dropped BOTH nginx and docker in favor of a more pure node-based setup. I have a feeling it's surfacing some errors that Docker was hiding.

**Reply by TechStudent10** [reply]
Maybe it should've been a good idea to release 0.5.6 as a beta to watch for stability problems, as it's a breaking change.

**Reply by benallfree** [reply]
@TechStudent10 Yeah, I sometimes have trouble reasoning through what a 'breaking' change means for pockethost. There were no API changes, but there was a significant infrastructure change that theoretically should not have broken anything (running the same code with the same config). But point taken. If you have any thoughts on a mental model that can help me navigate these decisions better, please share. I'd like for things to be stable by v1.0.

**Reply by TechStudent10** [reply]
@benallfree IMO, a breaking change is something the modifies functionality, under the hood or at the forefront. _(Note: this might not be the best way to think about it, but it works well for me.)_ In your case, you've replaced the building blocks of the app with something different. That would be a significant feature/modification to the program that there would be some bugs and errors, which is why I'd recommending maybe setting up a `beta` subdomain where features are tested for stability and UI purposes.
